usr.sbin/sysinst/msg.mi.pt

```diff
@@ -0,0 +1,1528 @@
+
+Dom�nio DNS: 		%s
+Nome do host: 		%s
+Servidor de nomes: 	%s
+Interface prim�ria:	%s
+Tipo de m�dia:		%s
+IP do host:			%s
+IP do servidor:		%s
+M�scara de rede:	%s
+Gateway IPv4:		%s
+}
+
+message netokv6
+{Configura�ao autom�tica IPv6:	%s
+}
+
+message netok_ok
+{Eles estao bem?}
+
+message wait_network
+{
+Aguardando enquanto a interface de rede � ativada.
+}
+
+message resolv
+{Nao foi poss�vel criar /etc/resolv.conf. Instala�ao abortada.
+}
+
+/* Called with: 			Example
+ *  $0 = target prefix			/target
+ *  $1 = error message			No such file or directory
+ */
+message realdir
+{Nao foi poss�vel alterar para o diret�rio $0: $1.
+Instala�ao abortada.}
+
+message delete_xfer_file
+{Excluir ap�s a instala�ao}
+
+/* Called with: 			Example
+ *  $0 = set name			base
+ */
+message notarfile
+{O conjunto de lan�amento $0 nao existe.}
+
+message endtarok
+{Todos os conjuntos de distribui�ao selecionados foram descompactados com sucesso.}
+
+message endtar
+{Houve problemas ao descompactar conjuntos de distribui�ao.
+Sua instala�ao est� incompleta.
+
+Voc� selecionou %d conjuntos de distribui�ao. %d conjuntos nao puderam ser encontrados
+e %d foram ignorados ap�s a ocorr�ncia de um erro. Dos %d
+que foram tentados, %d descompactaram sem erros e %d com erros.
+
+Abortando a instala�ao. Verifique novamente sua fonte de distribui�ao
+e considere reinstalar os conjuntos do menu principal.}
+
+message abort_inst {Instala�ao abortada.}
+message abort_part {Particionamento abortado.}
+
+message abortinst
+{A distribui�ao nao foi carregada com sucesso. Voc� precisar� prosseguir
+manualmente. Instala�ao abortada.
+}
+
+message abortupgr
+{A distribui�ao nao foi carregada com sucesso. Voc� precisar� prosseguir
+manualmente. Atualiza�ao abortada.
+}
+
+message abortunpack
+{Descompactar conjuntos adicionais nao foi bem-sucedido. Voc� precisar�
+continuar manualmente ou escolher uma fonte diferente para conjuntos de lan�amento e tentar
+novamente.
+}
+
+message createfstab
+{TH� um grande problema! Nao � poss�vel criar /mnt/etc/fstab. Saindo!
+}
+
+
+message noetcfstab
+{Socorro! Nenhum /etc/fstab no disco de destino %s. Abortando atualiza�ao.
+}
+
+message badetcfstab
+{Socorro! Nao � poss�vel analisar /etc/fstab no disco de destino %s. Abortando atualiza�ao.
+}
+
+message X_oldexists
+{Nao consigo salvar %s/bin/X como %s/bin/X.old, porque o
+disco de destino j� tem um %s/bin/X.old. Corrija isso antes
+de continuar.
+
+Uma maneira � iniciar um shell no menu Utilit�rios, examinar o
+destino %s/bin/X e %s/bin/X.old. Se
+%s/bin/X.old for de uma atualiza�ao conclu�da, voc� pode rm -f
+%s/bin/X.old e reiniciar. Ou se %s/bin/X.old for de
+uma atualiza�ao recente e incompleta, voc� pode rm -f %s/bin/X e mv
+%s/bin/X.old para %s/bin/X
+
+Abortando a atualiza�ao.}
+
+message netnotup
+{Houve um problema na configura�ao da rede. Ou seu gateway
+ou seu servidor de nomes nao estava acess�vel por um ping. Voc� quer
```

Comment:
I'm far from being an expert in Portuguese, but shouldn't "estava acessível" be in plural here, to match the gateway and name server?